python/pyspark/ml/classification.py

```diff
@@ -81,6 +160,8 @@ class LinearSVC(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol, Ha
     ...     Row(label=0.0, features=Vectors.dense(1.0, 2.0, 3.0))]).toDF()
     >>> svm = LinearSVC(maxIter=5, regParam=0.01)
     >>> model = svm.fit(df)
+    >>> model.setPredictionCol("prediction")
```

Comment:
What about change the value to a non-default value like "newPrediction", and make sure that the ouput dataframe/row has changed column name?